src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/NotificationPolicy.java

```diff
@@ -0,0 +1,34 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+/**
+ * NotificationPolicy encapsulates strategies to check and send notifications.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public abstract class NotificationPolicy<T> {
```

Comment:
no, just a normal interface, since you are defining more of an interface with the methods invoke (default method), check and send